app/lib/providers/onboarding_provider.dart

```diff
@@ -249,7 +249,12 @@ class OnboardingProvider extends BaseProvider with MessageNotifierMixin implemen
     ServiceManager.instance().device.subscribe(this, this);
 
     _findDevicesTimer = Timer.periodic(const Duration(seconds: 4), (timer) async {
-      ServiceManager.instance().device.discover();
+      if (deviceProvider != null && deviceProvider!.isConnected) {
+        _findDevicesTimer?.cancel();
```

Comment:
should we use 'timer' variable sir?

Reply:
We were using it from the very beginning, I guess it wasn't migrated to connectivity v1 or it was?

Reply:
very begining. btw i mean ![image](https://github.com/user-attachments/assets/5f848d8f-531a-40ca-8a99-6b902bc190c7)

Reply:
Oh I see, can be removed, since it isn't used

Reply:
Done

Reply:
haha use timer.cancel() i mean 🌚